src/render.js

```diff
@@ -3,37 +3,75 @@ function getPRComment(
   filesCoverage,
   minCoverageOverall,
   minCoverageChangedFiles,
-  title
+  title,
+  baselineData
 ) {
-  const fileTable = getFileTable(filesCoverage, minCoverageChangedFiles);
-  const overallTable = getOverallTable(overallCoverage, minCoverageOverall);
+  const fileTable = getFileTable(filesCoverage, minCoverageChangedFiles, baselineData);
   const heading = getTitle(title);
-  return heading + fileTable + `\n\n` + overallTable;
+  const description = getDescription(baselineData);
+  if (baselineData == null) {
+    const overallTable = getOverallTable(overallCoverage, minCoverageOverall);
+    return heading + description + fileTable + `\n\n` + overallTable;
+  } 
+  const overallTable = getOverallTableWithDelta(overallCoverage, baselineData.overallCoverage);
+  return heading + description + fileTable + `\n\n` + overallTable;
```

Comment:
interesting that there are template literal substitutions used throughout the file and then concatenations here. 

If this is meant to run on Node.js, then it is using v8 chromium which is slightly faster (1%) with template literal concatenation than string addition (SpiderMonkey is 1% faster with string addition). 

Just a curious observation, figure this isn't being run often enough for 1% perf gains to make a difference.